components/tidb_query/src/batch/executors/index_scan_executor.rs

```diff
@@ -183,8 +183,7 @@ impl ScanExecutorImpl for IndexScanExecutorImpl {
                 // This is a unique index, and we should look up PK handle in value.
 
                 // NOTE: it is not `number::decode_i64`.
-                value
-                    .read_i64::<BigEndian>()
+                ReadBytesExt::read_i64::<BigEndian>(&mut value)
```

Comment:
I discovered that maybe we can change to `value.read_u64(..).map(|v| v as i64)`, then `byteorder::{BigEndian, ReadBytesExt}` is no longer needed.

Reply:
👍 done